CHANGELOG.md

```diff
@@ -4,12 +4,25 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v2.9.0](https://github.com/lsst-it/puppet-ccs_sal/tree/v2.9.0) (2024-10-23)
+
+[Full Changelog](https://github.com/lsst-it/puppet-ccs_sal/compare/v2.8.1...v2.9.0)
+
+**Implemented enhancements:**
+
+- \(puppet-ccs\_sal\) bump saz/sudo version constraint to less than version 10.0.0 [\#56](https://github.com/lsst-it/puppet-ccs_sal/pull/56) ([badenerb](https://github.com/badenerb))
+
+## [v2.8.1](https://github.com/lsst-it/puppet-ccs_sal/tree/v2.8.1) (2024-08-30)
+
+[Full Changelog](https://github.com/lsst-it/puppet-ccs_sal/compare/v2.8.0...v2.8.1)
+
 ## [v2.8.0](https://github.com/lsst-it/puppet-ccs_sal/tree/v2.8.0) (2024-08-29)
 
 [Full Changelog](https://github.com/lsst-it/puppet-ccs_sal/compare/v2.7.0...v2.8.0)
 
 **Implemented enhancements:**
 
+- Version 2.8.0 [\#49](https://github.com/lsst-it/puppet-ccs_sal/pull/49) ([glennmorris](https://github.com/glennmorris))
```

Comment:
Could you change the label to `skip-changelog` for this PR and regenerate the changelog?  Someone reading the log probably already knows that the 2.8.0 release is the 2.8.0 release. :)

Reply:
<img width="391" alt="image" src="https://github.com/user-attachments/assets/3cc57bc7-3857-4e4e-9869-eb650b102f57">
I did add that label... perhaps I messed it up somehow 😆

Reply:
The label on this PR is correct but on #49 it is ![image](https://github.com/user-attachments/assets/c3d50496-42cb-46d5-b146-88975e5aaad7) when it should be `skip-changelog`.

Reply:
Fixed this by changing the tag on [#49 ](url)and regenerated the changelog